### PR TITLE
EPMRPP-93360 || Invite user modal

### PR DESCRIPTION
**Pull request by BlazarQSO**
## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx [hunk 1]
```diff
@@ -273,37 +312,42 @@ export class InviteUserModal extends Component {
       isAdmin,
       data: { isProjectSelector },
       projectKey,
+      projectName,
       areUserSuggestionsAllowed,
     } = this.props;
 
     const okButton = {
```
Comment (Vadim73i) [inline app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:319]: add 'data-automation-id': 'submitButton'

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx [hunk 2]
```diff
@@ -273,37 +312,42 @@ export class InviteUserModal extends Component {
...
-      onClick: (closeModal) => {
-        handleSubmit(this.inviteUserAndCloseModal(closeModal))();
+      children: intl.formatMessage(COMMON_LOCALE_KEYS.INVITE),
+      onClick: () => {
+        handleSubmit(this.inviteUserAndCloseModal)();
       },
       eventInfo: MEMBERS_PAGE_EVENTS.INVITE_BTN_INVITE_USER_MODAL,
     };
+
     const cancelButton = {
```
Comment (Vadim73i) [inline app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:327]: add 'data-automation-id': 'cancelButton',

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx [hunk 3]
```diff
@@ -20,56 +20,59 @@ import classNames from 'classnames/bind';
...
+import { hideModalAction, showModalAction } from 'controllers/modal';
 import { areUserSuggestionsAllowedSelector } from 'controllers/appInfo';
 import { AsyncAutocomplete } from 'components/inputs/autocompletes/asyncAutocomplete';
-import { InputDropdown } from 'components/inputs/inputDropdown';
 import { MEMBERS_PAGE_EVENTS } from 'components/main/analytics/events';
-import { projectKeySelector } from 'controllers/project';
+import { projectKeySelector, projectNameSelector } from 'controllers/project';
 import { InputUserSearch, makeOptions } from 'components/inputs/inputUserSearch';
 import { Input } from 'components/inputs/input';
+import { withTooltip } from 'componentLibrary/tooltip';
```
Comment (Vadim73i) [inline app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:45]: Can we use tooltip from ui-kit?

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx [hunk 4]
```diff
@@ -91,17 +124,19 @@ const inviteFormSelector = formValueSelector('inviteUserForm');
     selectedProject: ownProps.data.isProjectSelector
       ? inviteFormSelector(state, 'project')
       : urlProjectSlugSelector(state),
+    projectName: projectNameSelector(state),
     selectedUser: inviteFormSelector(state, 'user'),
     isAdmin: isAdminSelector(state),
     projectKey: projectKeySelector(state),
     initialValues: {
-      role: DEFAULT_PROJECT_ROLE,
+      role: false,
```
Comment (Vadim73i) [inline app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:132]: role is boolean?

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx [hunk 5]
```diff
@@ -226,14 +263,15 @@ export class InviteUserModal extends Component {
...
-  inviteUserAndCloseModal = (closeModal) => async (data) => {
+  inviteUserAndCloseModal = async (data) => {
     const {
       selectedProject,
       areUserSuggestionsAllowed,
       data: { isProjectSelector },
     } = this.props;
     const userData = {
       ...data,
+      role: data.role ? EDITOR : VIEWER,
```
Comment (Vadim73i) [inline app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:274]: Can each role edit?

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 1 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:319]: Done.

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 2 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:328]: Done.

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 3 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:45]: the tooltip in the ui-kit is not suitable

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 4 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:132]: roles are determined by the checkbox, false - Viewer, true - Editor

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 5 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:274]: EDITOR and VIEWER are roles

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 4 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx]: Done.

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 5 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx]: Done.

**Review by Vadim73i — APPROVED** [review]

**Pull request merged**